src/components/Panel/_mixins.scss

```diff
@@ -4,11 +4,11 @@
 /// @copyright IBM Security 2019
 ////
 
-@import '../../globals/index';
-
-@import '../../components/Header/mixins';
-@import '../../components/IconButton/mixins';
-@import '../../components/Toolbar/mixins';
+@import '@carbon/layout/scss/spacing';
```

Comment:
- [ ] We should also import `'@carbon/layout/scss/mini-unit'` if we're using `carbon--mini-units` directly in case `'@carbon/layout/scss/spacing'` stops referencing it in future - This should also apply to `Filter`, `InteractiveTag`, and `Tearsheet`

Reply:
- [ ] This change also needs to be applied to `Filter`, `InteractiveTag`, and `Tearsheet`